tests/integration/test_charm.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright 2024 Kimonas Sotirchos
+# See LICENSE file for licensing details.
+
+import asyncio
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./charmcraft.yaml").read_text())
+APP_NAME = METADATA["name"]
+
+
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test: OpsTest):
+    """Build the charm-under-test and deploy it together with related charms.
+
+    Assert on the unit status before any relations/configurations take place.
+    """
+    # Build and deploy charm from local source folder
```

Comment:
We could update those. I know it's a WIP but I 'm afraid after we commit those, there is a chance we miss updating this placeholder code

Reply:
ccabeab